dev/io.openliberty.io.opentelemetry.1.29/bnd.bnd

```diff
@@ -51,7 +51,7 @@ Export-Package: \
   io.opentelemetry.exporter.sender.okhttp.internal;version=${openTelemetryVersion};thread-context=true,\
   io.opentelemetry.exporter.zipkin.internal;version=${openTelemetryVersion};thread-context=true,\
   io.opentelemetry.sdk.autoconfigure.internal;version=${openTelemetryVersion};thread-context=true,\
-  io.opentelemetry.*;version=${openTelemetryVersion}
+  io.opentelemetry.*;version=${openTelemetryVersion};thread-context=true
```

Comment:
There has been no adequate explanation of why this is needed. It shouldn't be needed and should be removed ASAP.

This should not have been merged with this in it. At the very least you can take out all of the lines above it because they only exist to put specific packages on the thread context classloader.

Reply:
If there are _specific_ packages which need to be added to the TCCL, that's fine, but adding everything with no explanation is not.

Reply:
Raised #26313 to do this.

Reply:
It's also not clear why this wasn't needed before but is now.

If it was needed before, then this would have been a release bug and there needs to be an issue raised for that.

Reply:
Although, in terms of raising a release bug, I'm more concerned with the change to the 1.0 project.